qiskit_ibm_runtime/ibm_runtime_service.py

```diff
@@ -537,127 +478,74 @@ def _aliased_backend_names() -> Dict[str, str]:
-            `hub`, `group`, `project` must be specified.
-
         Args:
-            token: IBM Quantum token.
-            url: URL for the IBM Quantum authentication server.
-            hub: Name of the hub.
-            group: Name of the group.
-            project: Name of the project.
-            overwrite: Overwrite existing credentials.
-            **kwargs:
-                * proxies (dict): Proxy configuration for the server.
-                * verify (bool): If False, ignores SSL certificates errors
-
-        Raises:
-            IBMProviderCredentialsInvalidUrl: If the `url` is not a valid
-                IBM Quantum authentication URL.
-            IBMProviderCredentialsInvalidToken: If the `token` is not a valid
-                IBM Quantum token.
-            IBMProviderValueError: If only one or two parameters from `hub`, `group`,
-                `project` are specified.
+            auth: Authentication type. `cloud` or `legacy`.
+            token: IBM Cloud API key or IBM Quantum token.
+            name: Custom name of the saved account. Defaults to "default".
+            url: The API URL.
+                Defaults to https://cloud.ibm.com (cloud) or https://auth.quantum-computing.ibm.com/api (legacy).
+            instance: The CRN (cloud) or provider string (legacy).
+            proxies: Proxy configuration for the server.
+            verify: If False, ignores SSL certificates errors
         """
-        if url != QISKIT_IBM_RUNTIME_API_URL:
-            raise IBMProviderCredentialsInvalidUrl(
-                "Invalid IBM Quantum credentials found."
-            )
-        if not token or not isinstance(token, str):
-            raise IBMProviderCredentialsInvalidToken(
-                "Invalid IBM Quantum token "
-                'found: "{}" of type {}.'.format(token, type(token))
-            )
-        # If any `hub`, `group`, or `project` is specified, make sure all parameters are set.
-        if any([hub, group, project]) and not all([hub, group, project]):
-            raise IBMProviderValueError(
-                "The hub, group, and project parameters must all be "
-                "specified when storing a default hub/group/project to "
-                'disk: hub = "{}", group = "{}", project = "{}"'.format(
-                    hub, group, project
-                )
-            )
-        # If specified, get the hub/group/project to store.
-        default_hgp_id = (
-            HubGroupProjectID(hub, group, project)
-            if all([hub, group, project])
-            else None
-        )
-        credentials = Credentials(
-            token=token, url=url, default_provider=default_hgp_id, **kwargs
+
+        AccountManager.save(
+            name=name,
+            account=Account(
+                auth=auth,
+                url=url,
+                token=token,
+                instance=instance,
+                proxies=proxies,
+                verify=verify,
+            ),
```

Comment:
Same here, it’d be nice if AccountManager knew how to create Account object vs IBMRuntimeService having to know that. Code can be simpler here as well:

AccountManager.save(name, auth, url, …)

Reply:
Since that's an implementation detail I hope an additional class is ok to use. If you have a very strong opinion about changing this, I can do it.

My personal preference is to keep it, though. I find the current interface cleaner than passing on primitive values another nesting level. Also, this way `AccountManager.save` and `AccountManager.get` have a very clear relationship. 

```
def save(account: Account, name: Optional[str] = _DEFAULT_ACCOUNT_NAME) -> None:
def get(name: Optional[str] = _DEFAULT_ACCOUNT_NAME) -> Account:
```

Reply:
Thought a bit more about it (while working on unit tests in the follow-up) and start seeing benefits of putting account creation logic to the account manager. 

Addressed your feedback via https://github.com/Qiskit/qiskit-ibm-runtime/pull/49/commits/0c31c42182812b2b102a9f9116822862ae39e5bd